modules/microshift-configuring-ovn.adoc

```diff
@@ -71,4 +70,11 @@ mtu: 1300
 ----
 
 [IMPORTANT]
+====
 When `disableOVSInit` is set to true in the `ovn.yaml` config file, the OVS bridge br-ex must be manually configured.
+====
+
+[IMPORTANT]
+====
+If you change the `mtu` configuration value in the `ovn.yaml` file, you must restart the ovnkube-master pod and the host that {product-title} is running on for for the updated setting to apply.
```

Comment:
```suggestion
If you change the `mtu` configuration value in the `ovn.yaml` file, you must reboot the host that {product-title} is running on for the updated setting to apply.
```

Reply:
We don't use "reboot" per IBM style, but I got the rest.

Reply:
I see, l wasn't aware of the IBM style. Let's use restart.